packages/core/src/model/Project.ts

```diff
@@ -12,6 +12,7 @@ export type Project = {
     title: string;
     description: string;
     mainGraphId?: GraphId;
+    path?: string;
```

Comment:
This is going to serialize the path of the project in the project itself, which I'd like to avoid :/

Can you change the project metadata serialization to ignore this?

Reply:
pushed a change!